src/ToricVarieties/cohomCalg/auxiliary.jl

```diff
@@ -96,10 +96,14 @@ function contributing_denominators(variety::AbstractNormalToricVariety)
     # -> translate back into actually chosen variable names
     gens = Hecke.gens(cox_ring(variety))
     for i in 1:length(contributing_monomials)
```

Comment:
off-topic, but: why don't you do this instead:
```suggestion
    for cm in contributing_monomials
```
and then replace each occurrence of `contributing_monomials[i]` inside the loop by just `cm`?